docs/src/test/java/docs/http/javadsl/server/testkit/MyAppFragment.java

```diff
@@ -3,22 +3,17 @@
  */
 
 package docs.http.javadsl.server.testkit;
-//#source-quote
+// #source-quote
 import akka.http.javadsl.server.AllDirectives;
 import akka.http.javadsl.server.Route;
 
 public class MyAppFragment extends AllDirectives {
 
-    public Route createRoute() {
-        return
-                //#fragment
-                pathEnd(() ->
-                        get(() ->
-                                complete("Fragments of imagination")
-                        )
-                );
-                //#fragment
-    }
-
+  public Route createRoute() {
+    return
+    // #fragment
+    pathEnd(() -> get(() -> complete("Fragments of imagination")));
```

Comment:
Cramming routes on one line like this really isn't nice, so that alone is a reason not to use sbt-java-formatter I think (though some of the other proposed changes are nice).

Reply:
Time to fork https://github.com/google/google-java-format (joking)

Reply:
The collapse of routes DSL into a single line is unfortunate and won't pay off for the rest of improvements. :-(

Reply:
We could try to use it for `compile` scope only, assuming that most of the DSL use is in `test` scope.

Reply:
Not sure if it's worth it then...